pkg/k3s/k3s.go

```diff
@@ -51,17 +51,24 @@ func StartK3S(ctx context.Context, vConfig *config.VirtualClusterConfig, service
 			args = append(args, "--kube-apiserver-arg=endpoint-reconciler-type=none")
 		}
 		if vConfig.ControlPlane.BackingStore.Etcd.Deploy.Enabled {
+			var etcdEndpoints string
```

Comment:
NIT: Given that we only have a single endpoint here, this might become confusing.

```suggestion
			var etcdEndpoint string
```

Reply:
I just tried to match it with the name of the function parameter where it is used and also other [places](https://github.com/neogopher/vcluster/blob/ce4030a9679d4880f73a1884e1abf0857429a489/pkg/etcd/client.go#L66-L67) where it is exists as of today, but I agree it is largely singular in our code.

Have updated just the ones I added.